test/plots/rectanglePlotTests.ts

```diff
@@ -2,511 +2,519 @@
 
 describe("Plots", () => {
   describe("RectanglePlot", () => {
-    let SVG_WIDTH = 300;
-    let SVG_HEIGHT = 300;
-    let DATA = [
-      { x: 0, y: 0, x2: 1, y2: 1 },
-      { x: 1, y: 1, x2: 2, y2: 2 },
-      { x: 2, y: 2, x2: 3, y2: 3 },
-      { x: 3, y: 3, x2: 4, y2: 4 },
-      { x: 4, y: 4, x2: 5, y2: 5 }
-    ];
-    let VERIFY_CELLS = (cells: d3.Selection<any>) => {
-      assert.strictEqual(cells[0].length, 5);
-      cells.each(function(d: any, i: number) {
-        let cell = d3.select(this);
-        assert.closeTo(+cell.attr("height"), 50, 0.5, "Cell height is correct");
-        assert.closeTo(+cell.attr("width"), 50, 0.5, "Cell width is correct");
-        assert.closeTo(+cell.attr("x"), 25 + 50 * i, 0.5, "Cell x coordinate is correct");
-        assert.closeTo(+cell.attr("y"), 25 + 50 * (cells[0].length - i - 1), 0.5, "Cell y coordinate is correct");
+    describe("Basic usage", () => {
+      it("has correct rectangle heights and positions (renders correctly)", () => {
+        let data = [
+          { x: 0, y: 0, x2: 1, y2: 1 },
+          { x: 1, y: 1, x2: 2, y2: 2 },
+          { x: 2, y: 2, x2: 3, y2: 3 },
+          { x: 3, y: 3, x2: 4, y2: 4 },
+          { x: 4, y: 4, x2: 5, y2: 5 }
+        ];
+
+        let svg = TestMethods.generateSVG(300, 300);
+        let xScale = new Plottable.Scales.Linear();
+        let yScale = new Plottable.Scales.Linear();
+        let plot = new Plottable.Plots.Rectangle<number, number>();
+        plot.x((d) => d.x, xScale).x2((d) => d.x2);
+        plot.y((d) => d.y, yScale).y2((d) => d.y2);
+        plot.addDataset(new Plottable.Dataset(data));
+        plot.renderTo(svg);
+
+        let cells = plot.content().selectAll("rect");
+        assert.strictEqual(cells.size(), data.length);
+        cells.each(function(d, i) {
+          let cell = d3.select(this);
+          assert.closeTo(+cell.attr("height"), 50, 0.5, "Cell height is correct");
+          assert.closeTo(+cell.attr("width"), 50, 0.5, "Cell width is correct");
+          assert.closeTo(+cell.attr("x"), 25 + 50 * i, 0.5, "Cell x coordinate is correct");
+          assert.closeTo(+cell.attr("y"), 25 + 50 * (cells[0].length - i - 1), 0.5, "Cell y coordinate is correct");
```

Comment:
Unsure but I think we want to use window.Pixel_CloseToRequirement here instead of 0.5